File: 2.0/csharp/contacts/getDoNotContact.cs
using CallfireApiClient;
using CallfireApiClient.Api.Contacts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        DoNotContact dnc = client.DncApi.Get("12135551126");
    }
}

File: 2.0/java/contacts/getDoNotContact.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.contacts.model.DoNotContact;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        DoNotContact dnc = client.dncApi().get("12135551126");
    }
}
